src/layout/Alert/types.d.ts

```diff
@@ -0,0 +1,10 @@
+import type { ILayoutCompBase } from 'src/layout/layout';
+
+export type AlertSeverity = 'success' | 'warning' | 'danger' | 'info';
+
+export interface ILayoutCompAlertBase {
+  severity: AlertSeverity;
+  useAsAlert?: boolean;
```

Comment:
I feel this property maybe needs a new name as the components name is `Alert`. 🤔 What about `useAriaLive` or something similar?

Side thought that might not work: Maybe it is possible to check if the Alert components `required` property in the layout file contains a boolean value or an expression(which is written as a string) and set this value to `true` if an expression is used?

Reply:
Agree! 💯 But I assume you mean the `hidden` property? An `Alert` is not a form component, so it cannot have any functionality for `required`.

Reply:
@olemartinorg, Yes thats what i meant! Sorry 😄

Reply:
Yeah that seems like a good idea. I'll see if that is possible :sweat_smile: